src/modules/jcms_notifications/src/NotificationStorageInterface.php

```diff
@@ -18,31 +18,31 @@
    *
    * @return int|null
    */
-  public function saveNotificationNid(EntityInterface $entity);
+  public function saveNotificationEntityId(EntityInterface $entity);
 
   /**
    * Gets the node IDs from the notifications table then deletes them.
```

Comment:
Not sure how I missed these last time! Remove node from the method annotations in this file.